### DISCUSSION TITLE
How to add service worker without any dependencies like next-offline package?

### DISCUSSION THREAD

**Discussion by melvin2016** [vercel/next.js#12773] [Help] [answered]
I tried to register service worker in _app.js page.but it doesnt seem to update when making changes?

Is there any way @timneutkens

**Comment by leerob** [comment, marked as answer]
Right now, `next-offline` is the easiest way to add offline support. Without a reproducible example, it's hard to help debug why your application isn't working. Could you provide a CodeSandbox link?

**Reply by melvin2016** [reply]
Yeah I got it to work with the next-offline package. Thanks.
